controllers/packetmachine_controller.go

```diff
@@ -175,13 +175,13 @@ func (r *PacketMachineReconciler) reconcile(ctx context.Context, machineScope *s
 
 	if !machineScope.Cluster.Status.InfrastructureReady {
 		machineScope.Info("Cluster infrastructure is not ready yet")
-		return ctrl.Result{Requeue: true, RequeueAfter: 30 * time.Second}, nil
+		return ctrl.Result{}, nil
```

Comment:
Why do we no longer requeue? Don't we need to tell it to come back and queue up the request again? Or does the Machine controller change something on the PacketMachine when the cluster is ready? Something has to do it.

Reply:
copied it from aws. It works